crates/e2e/tests/e2e/protocol_fee.rs

```diff
@@ -131,6 +139,37 @@ async fn volume_fee_sell_order_test(web3: Web3) {
         web3.clone(),
         fee_policy,
         OrderKind::Sell,
+        None,
+        1000150353094783059u128.into(),
+        987306456662572858u128.into(),
+    )
+    .await;
+}
+
+async fn partner_fee_sell_order_test(web3: Web3) {
+    // Fee policy to be overwritten by the partner fee
+    let fee_policy = FeePolicyKind::Volume { factor: 0.5 };
```

Comment:
I used here a factor of `0.5` and in the partner fee a factor of `0.1` (10 bps) in order to test if it really overwrites the fee policy.